sqlserver/datadog_checks/sqlserver/data/conf.yaml.example

```diff
@@ -120,6 +120,7 @@ instances:
 ##                                         Set channel_path if type is windows_event.
 ## source  - required - Attribute that defines which Integration sent the logs.
 ## service - optional - The name of the service that generates the log.
+## encoding - optional - Set the file encoding, SQL Server error log uses utf-16-le.
```

Comment:
Instead of directly changing the conf.yaml.example, you could modify the spec.yaml file (sqlserver/assets/configuration/spec.yaml) since we run a script to automatically generate the conf.yaml.example now. There is a template for logs (datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/logs.yaml), which you can just use and override to include the additional `encoding` information.